en/core-libraries/email.rst

```diff
@@ -535,6 +535,11 @@ following::
         }
     }
 
+.. deprecated:: 3.4.0
```

Comment:
Can you just update the example here instead?

You can put a comment above the function to indicate when it was deprecated if you want.
// Prior to 3.4, use template()